### PR TITLE
Nas 103706

### PR DESCRIPTION
**Pull request by dkmullen**
Removes rest references and notifiers from bulk disk edit page; adds fieldset

### CODE REVIEW

File: src/app/pages/storage/disks/disk-bulk-edit/disk-bulk-edit.component.ts [hunk 1]
```diff
@@ -16,74 +18,88 @@ import { StorageService } from '../../../../services/storage.service'
...
-    },
-    {
-      type: 'input',
-      name: 'disk_smartoptions',
-      placeholder: T('S.M.A.R.T. extra options'),
-      value: this.diskBucket.SMARToptions,
-      tooltip : T('Additional <a\
-                   href="https://www.smartmontools.org/browser/trunk/smartmontools/smartctl.8.in"\
-                   target="_blank">smartctl(8)</a> options.'),
+      name: 'Disks',
```
Comment (erinix) [inline src/app/pages/storage/disks/disk-bulk-edit/disk-bulk-edit.component.ts]: Needs a T(), should probably be in helptext

File: src/app/pages/storage/disks/disk-bulk-edit/disk-bulk-edit.component.ts [hunk 2]
```diff
@@ -16,74 +18,88 @@ import { StorageService } from '../../../../services/storage.service'
...
+          type: 'input',
+          name: 'disk_name',
+          placeholder: T('Editing the following disks:'),
+          tooltip : T('This is the FreeBSD device name for the disk.'),
+          value: [this.diskBucket.diskNames],
+          readonly: true
+        }
+      ]
+    }, {
+      name: 'Settings',
```
Comment (erinix) [inline src/app/pages/storage/disks/disk-bulk-edit/disk-bulk-edit.component.ts]: Also should have a T()

**Comment by erinix** [comment]
@freenas/docs-team could you review the tooltips and labels?

File: src/app/helptext/storage/disks/disk-form.ts [hunk 3]
```diff
@@ -56,4 +85,21 @@ disk_form_passwd_tooltip: T('Password for SED'),
 
 disk_form_passwd2_placeholder: T('Confirm SED Password'),
 disk_form_passwd2_tooltip: T(''),
+
+bulk_edit: {
+    title: T('Disks'),
+    label: T('Settings'),
+    disks : {
+        placeholder: T('Editing the following disks:'),
```
Comment (a-stjohn) [inline src/app/helptext/storage/disks/disk-form.ts]:
```suggestion
        placeholder: T('Disks to be edited:'),
```

File: src/app/helptext/storage/disks/disk-form.ts [hunk 4]
```diff
@@ -56,4 +85,21 @@ disk_form_passwd_tooltip: T('Password for SED'),
...
+        tooltip : T('These are the FreeBSD device names for each disk.'),
```
Comment (a-stjohn) [inline src/app/helptext/storage/disks/disk-form.ts]:
```suggestion
        tooltip : T('FreeBSD device names of each disk being edited.'),
```

File: src/app/helptext/storage/disks/disk-form.ts [hunk 5]
```diff
@@ -56,4 +85,21 @@ disk_form_passwd_tooltip: T('Password for SED'),
...
+    },
+    serial : {
+        placeholder: T('Serial'),
+        tooltip : T('These are the serial numbers for each disk.')
```
Comment (a-stjohn) [inline src/app/helptext/storage/disks/disk-form.ts]:
```suggestion
        tooltip : T('Serial numbers of each disk being edited.')
```

File: src/app/helptext/storage/disks/disk-form.ts [hunk 6]
```diff
@@ -56,4 +85,21 @@ disk_form_passwd_tooltip: T('Password for SED'),
...
+    }
+
+}, 
+dialog_error: T('Error updating disks')
```
Comment (a-stjohn) [inline src/app/helptext/storage/disks/disk-form.ts]: Is there anyway for this error to be more informative to the user?

File: src/app/helptext/storage/disks/disk-form.ts [hunk 7]
```diff
@@ -18,13 +18,42 @@ disk_form_hddstandby_tooltip : T('Indicates the time of inactivity in minutes be
...
+ {label:'120', value: '120'},
+ {label:'180', value: '180'},
+ {label:'240', value: '240'},
+ {label:'300', value: '300'},
+ {label:'330', value: '330'},
+],
+
 disk_form_advpowermgmt_placeholder: T('Advanced Power Management'),
 disk_form_advpowermgmt_tooltip : T('Select a power management profile from the menu.'),
+disk_form_advpowermgmt_options: [
```
Comment (erinix) [inline src/app/helptext/storage/disks/disk-form.ts]: we should add T() to these things

File: src/app/helptext/storage/disks/disk-form.ts (hunk 6 above)
Comment (dkmullen) [reply src/app/helptext/storage/disks/disk-form.ts]: This is just the title of the dialog; there should also be an error message from the middleware

**Pull request merged**